README.md

```diff
@@ -8,15 +8,28 @@
 
 </div>
 
-This page is available as an easy-to-read website at [![https://ebookfoundation.github.io/free-programming-books/](https://img.shields.io/website?style=flat&logo=www&logoColor=whitesmoke&label=https%3A%2F%2Febookfoundation.github.io&down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Febookfoundation.github.io%2Ffree-programming-books%2F)](https://ebookfoundation.github.io/free-programming-books/)
+This page is available as an easy-to-read website at [![https://ebookfoundation.github.io/free-programming-books/](https://img.shields.io/website?style=flat&logo=www&logoColor=whitesmoke&label=https%3A%2F%2Febookfoundation.github.io&down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Febookfoundation.github.io%2Ffree-programming-books%2F)](https://ebookfoundation.github.io/free-programming-books/). Also try our new dynamic version of the website at [![https://ebookfoundation.github.io/free-programming-books-search/](https://img.shields.io/website?style=flat&logo=www&logoColor=whitesmoke&label=https%3A%2F%2Febookfoundation.github.io/free-programming-books-search&down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Febookfoundation.github.io%2Ffree-programming-books-search%2F)](https://ebookfoundation.github.io/free-programming-books-search/). 
 
+<div align="center">
+  <form action="https://ebookfoundation.github.io/free-programming-books-search">
+    <!-- <label for="search">Search: </label> -->
+    <input type="text" id="fpbSearch" name="search" placeholder="Search Book or Author"/>
+    <label for="submit"> </label>
+    <input type="submit" name="submit" value="Search" />
+  </form>
```

Comment:
attribute `for` always goes against target element `id`. Another way to solve without ids is wrapping the `input` into the `label`

```suggestion

  <form action="https://ebookfoundation.github.io/free-programming-books-search">
    <!-- <label for="fpbSearch">Search: </label> -->
    <input type="text" id="fpbSearch" name="search" placeholder="Search Book or Author"/>
    <label for="submit"> </label>
    <input type="submit" id="submit" name="submit" value="Search" />
  </form>

```

Reply:
Added the id and removed the commented out label